mod_reforged/hooks/misc.nut

```diff
@@ -4,3 +4,65 @@
 	if (_function != null) _function(obj);
 	return obj;
 }
+
+// Return a list with ids of all weapon perk groups belonging to the given _weapon
+::Reforged.getWeaponPerkGroups <- function( _weapon )
+{
+	local ids = [];
+
+	if (_weapon == null || !_weapon.isItemType(::Const.Items.ItemType.Weapon))
```

Comment:
Not sure if we should allow passing null here. It's more convenient, but kind of delays issues.

Reply:
I remove that check and emphasize in the documentation that the caller needs to do that check